_includes/integrations/react-tech-ref.md

```diff
@@ -201,13 +309,55 @@ Used to specify the format of the content emitted via the [`onEditorChange`](#on
 ##### Example: Using `outputFormat`
 
 ```jsx
-<Editor
-  outputFormat='text'
-/>
+const textToHtml = (text) => {
+  const elem = document.createElement('div');
+  return text.split(/\n\n+/).map((paragraph) => {
+    return '<p>' + paragraph.split(/\n+/).map((line) => {
+      elem.textContent = line;
+      return elem.innerHTML;
+    }).join('<br/>') + '</p>';
+  }).join('');
+};
+const initialText = 'The quick brown fox jumps over the lazy dog';
+const [text, setText] = useState(initialText);
+return (
+  <>
+    <Editor
+      initialValue={textToHtml(initialText)}
+      outputFormat='text'
+      onEditorChange={(newText) => setText(newText)}
+    />
+    <pre>{text}</pre>
+  </>
+);
+```
+
+##### Example: Replacing usage of `outputFormat`
+
+```jsx
+const [value, setValue] = useState('<p>The quick brown fox jumps over the lazy dog</p>');
+const [text, setText] = useState('');
+
+return (
+  <>
+    <Editor
+      value={value}
+      onInit={(evt, editor) => {
+        setText(editor.getContent({format: 'text'}));
+      }}
+      onEditorChange={(newValue, editor) => {
+        setValue(newValue);
+        setText(editor.getContent({format: 'text'}));
+      }}
+    />
+    <pre>{text}</pre>
+  </>
+);
 ```
 
 #### `plugins`
-Used to include plugins for the editor. Using `<Editor plugins='lists' />` is the same as setting `{plugins: 'lists'}` in the {{site.productname}} selector (`tinymce.init`).
+
+Used to include plugins for the editor. Using `<Editor plugins='lists' />` is the same as setting `{plugins: 'lists'}` in the {{site.productname}} method `tinymce.init({...})`.
```

Comment:
```suggestion
Used to include plugins for the editor. Using `<Editor plugins='lists' />` is the same as setting `{plugins: 'lists'}` in the {{site.productname}} `tinymce.init({...})` method.
```
Maybe? It just reads rather odd at the moment is all.

Reply:
Hmmm, I'm not sure I agree it's better. If we put in the product name it reads:
> in the TinyMCE method `tinymce.init({...})`.

With your change it becomes:
> in the TinyMCE `tinymce.init({...})` method.

which seems worse to me.

Reply:
@tylerkelly13 which version would you prefer?

Reply:
we could always drop the `productname` in the above as well so that it becomes:
> in the tinymce.init({...}) method.

Reply:
I guess the reason it's spelled out like that is that the react users may not have used TinyMCE directly before so they may not have ever seen the `init` method before.

Reply:
It's written that way in 4 places. Twice in the `init` description, once in `inline` and once in `plugins`

Reply:
Though it is additionally phrased "the `tinymce.init({...})` method" in `init`